src/main/java/de/retest/web/ScreenshotProvider.java

```diff
@@ -12,20 +14,33 @@
 
 	private static final int SCROLL_TIMEOUT = 100;
 	private static final boolean USE_DEVICE_PIXEL_RATIO = true;
+	private static final int WINDOW_WIDTH = 1200;
 
 	private ScreenshotProvider() {
 		// private constructor for util class
 	}
 
 	public static BufferedImage shootFullPage( final WebDriver driver ) {
+		final BufferedImage image;
 		if ( driver instanceof ChromeDriver ) {
-			return Shutterbug
+			image = Shutterbug
 					.shootPage( driver, ScrollStrategy.WHOLE_PAGE_CHROME, SCROLL_TIMEOUT, USE_DEVICE_PIXEL_RATIO )
 					.getImage();
+			return resize( image, WINDOW_WIDTH, image.getData().getHeight() / 2 );
```

Comment:
Why `/2`? Shouldn't a fixed width do the trick? Also, a fixed divisor works probably only with retina displays...

Reply:
This update is only required if you want to take a screenshot with the Chromedriver on a Retina MacBook. For example. showcaseIT-chromedriver ... without resizing we get a width of ~ 2300px and height ~ 6200px. We are resizing to 1200px, but still have a height of 6200px and that looks awful